suzieq/poller/worker/nodes/node.py

```diff
@@ -1863,18 +1868,23 @@ async def _rest_gather(self, service_callback, cmd_list, cb_token,
 
     async def _fetch_init_dev_data_devtype(self, reconnect: bool):
         """Fill in the boot time of the node by running requisite cmd"""
-        await self._exec_cmd(self._parse_init_dev_data,
-                             ["show system uptime|display json",
-                              "show version"], None, 'mixed',
-                             reconnect=reconnect)
+        if self.devtype == 'junos-evo':
+            cmdlist = ["show system uptime|display json",
+                       'show version node all']
+        else:
+            cmdlist = ["show system uptime|display json",
+                       'show version']
```

Comment:
@ddutt why don't create a new class for junos evo instad of doing this if else? It seems a cleaner solution

Reply:
Because we'll be duplicating a lot of work